include/core/utilities.h

```diff
@@ -585,4 +585,32 @@ value_string_to_tensor(const std::string &value_string_0,
   return output_tensor;
 }
 
+/**
+ * @brief Computes equivalent cell diameter by comparing the area to a disk (2D)
+ * or the volume to a sphere (3D).
+ *
+ * @tparam dim Number of spatial dimensions (2D or 3D).
+ *
+ * @param[in] cell_measure Area (2D) or volume (3D) of the cell.
+ *
+ * @param[in] fe_degree Polynomial degree of the shape function.
+ *
+ * @return Cell diameter value.
+ */
+template <int dim>
+inline double
+compute_cell_diameter(const double cell_measure, const unsigned int fe_degree)
+{
+  double h;
+  if constexpr (dim == 2)
+    h = std::sqrt(4. * cell_measure / numbers::PI) / fe_degree;
+  else if constexpr (dim == 3)
+    h = std::pow(6. * cell_measure / numbers::PI, 1. / 3.) / fe_degree;
+  else
+    throw std::invalid_argument(
+      "'dim' should have a value of either 2 or 3. Only 2D and 3D simulations "
+      "are supported.");
```

Comment:
The Assert() macro here should be used and not a throw statement. This I also suggested in my previous comments.
```suggestion
    Assert(false, ExcMessage(std::string(
      "'dim' should have a value of either 2 or 3. Only 2D and 3D simulations "
      "are supported."));
```
Something like this should be the syntax. There's nothing to throw here, we won't recover from that situation.

Reply:
I wasn't sure of what to put as a condition and was confused, but this totally makes sens!